packages/integrations/cloudflare/README.md

```diff
@@ -92,16 +92,17 @@ To do this:
 
 ## Environment Variables
 
-As Cloudflare Pages Functions [provides environment variables differently](https://developers.cloudflare.com/pages/platform/functions/#adding-environment-variables-locally), private environment variables needs to be set through [`vite.define`](https://vitejs.dev/config/shared-options.html#define) to work in builds.
+As Cloudflare Pages Functions [provides environment variables per request](https://developers.cloudflare.com/pages/platform/functions/#adding-environment-variables-locally), private environment variables should only be accessed when a request has happened. Usually, this means moving environment variable access inside a function.
```

Comment:
Hmm using a function is not the only way to workaround this issue, so it feels a bit odd to directly say "use a function". You can use `import.meta.env.PRIVATE_VAR` in an `.astro` file without a function for example.

I'm also not sure how to explain this well as a "request" is generally a Cloudflare implementation detail, so I wrote this as "if you see this issue, try this solution". I'm still slightly leaning on the original because of this though. 😅

Reply:
OK, yah, if that's not the only solution, makes sense!

Can you revisit the original re: the word "should" though? That's a tricky word. It can be really difficult to tell whether "private environment variables should only be accessed when a request has happened." means:

You must take care to only access them when..(best practice) 

vs.

They are likely only going to be accessed (probability/what's likely to happen)

It was really that one single word that made me think maybe we could get clearer with a restructuring!

Reply:
Good point, perhaps something like below could be more assertive:

```suggestion
As Cloudflare Pages Functions [provides environment variables per request](https://developers.cloudflare.com/pages/platform/functions/#adding-environment-variables-locally), you can only access private environment variables when a request has happened. Usually, this means moving environment variable access inside a function.
```

Reply:
I like it! Ship it! 🚀